cli/cmd/create_project.go

```diff
@@ -112,14 +127,38 @@ keptn create project PROJECTNAME --git-user=GIT_USER --git-remote-url=GIT_REMOTE
 			project.GitToken = *createProjectParams.GitToken
 			project.GitRemoteURL = *createProjectParams.RemoteURL
 
+			if *createProjectParams.GitProxyURL != "" && strings.HasPrefix(*createProjectParams.RemoteURL, "ssh://") {
+				return errors.New(gitErrMsg)
```

Comment:
I think we need to be more specific with the error message here, telling the user why ssh:// cannot be used as a prefix